Al_standard/imported.expt

```diff
@@ -0,0 +1,204 @@
+{
+  "__id__": "ExperimentList",
+  "experiment": [
+    {
+      "__id__": "Experiment",
+      "identifier": "570c385d-dc55-4aa7-8064-9739ebc06a1c",
+      "beam": 0,
+      "detector": 0,
+      "goniometer": 0,
+      "scan": 0,
+      "imageset": 0
+    }
+  ],
+  "imageset": [
+    {
+      "__id__": "ImageSequence",
+      "template": "/home/elena/Work/Diamond/Data/data-files/Al_standard/0p67_5s_####.mrc",
```

Comment:
reference to absolute path

Reply:
I think you just edit the path in the file to be relative and it will work fine

Reply:
It seems that if I call `dials.import template=file.mrc` it won't save the path in `imported.expt`

Reply:
Yeah I don't think there's any way other than getting down and dirty with a text editor